.gitignore

```
@@ -1,2 +1,2 @@
-/zig-cache
-/zig-out
+.zig-cache/
+zig-out/
```

```suggestion
/zig-out/
```

The preceding slash means it only matches at the root of the repo. The trailing slash means it only matches directories.